dbms/src/DataStreams/tests/collapsing_sorted_stream.cpp

```diff
@@ -30,13 +30,13 @@ try
         ColumnWithTypeAndName column1;
         column1.name = "Sign";
         column1.type = std::make_shared<DataTypeInt8>();
-        column1.column = ColumnInt8::create({1, -1});
+        column1.column = ColumnInt8::create({static_cast<int8_t>(1), static_cast<int8_t>(-1)});
```

Comment:
Why don't make the type of initializer list generic in ColumnVector and PODArray constructors? Let me try to do it in subsequent PR...